src/cli/tools/dotnet.ts

```diff
@@ -116,23 +116,9 @@ export class DotnetInstallService extends BaseInstallService {
       file,
       cwd: toolPath,
       strip: 1,
+      options: ['--uid', `${this.envSvc.userId}`, '--gid', '0'],
```

Comment:
@rarkins i wonder if we should set this as default for extracts, so files are always owned by our default user 🤔